### PR TITLE
Update stories to use new inputProvider

### LINKED ISSUE #495: InputProvider errors on components with children
**Describe the bug**
When you pass a component that has a `child` or `children` to `inputProvider`, an error is thrown.
```jsx
InputProvider(['inline link', 'caption'], (linkText, captionText) => (
  <Caption script={latin}>
    {captionText}
    <InlineLink href="https://www.bbc.com"> {linkText} </InlineLink>
  </Caption>
)
```

**To Reproduce**

1. Go to Storybook Caption
2. Click on with offscreen text

**Expected behavior**
`inputProvider` should be able to handle components with and without children.

**Screenshots**
**Desktop (please complete the following information):**

- OS: macOs
- Browser: Chrome
- Version: 74

**Smartphone (please complete the following information):**

**Additional context**

### LINKED ISSUE #497: Update stories to use @bbc/psammead-storybook-helpers@2.0.0
**Is your feature request related to a problem? Please describe.**
Some stories on storybook throw errors because of a recent update to `inputProvider` where its API changed

**Describe the solution you'd like**
Update `Caption`, `SubHeading`,  `SitewideLinks`, `InlineLink`, `VisuallyHiddenText` and `StoryPromo` stories to use the new `inputProvider`

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Visual inspection is sufficient

**Additional context**

### PR DESCRIPTION
**Pull request by ghost**
Resolves #497 
Resolves #495

**Overall change:** Update stories that use `inputProvider` to use version `2.0.0`

**Code changes:**

- Updated `Caption`, `SubHeading`, `SitewideLinks`, `InlineLink`, `VisuallyHiddenText` and `StoryPromo` stories to use `@bbc/psammead-storybook-helpers@2.0.0`

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Review by dr3 — CHANGES_REQUESTED** [review]
Can you remove the `defaultText` key from all stories except `packages/components/psammead-sitewide-links/src/index.stories.jsx` please. Using the english translation is nicer than `'headline'` or `'caption'`

**Review by dr3 — CHANGES_REQUESTED** [review]
Pr has conflict which will need resolving

**Review by DenisHdz — DISMISSED** [review]
LGTM 👍

**Review by dr3 — CHANGES_REQUESTED** [review]
We need to do Paragraph too, https://github.com/bbc/psammead/blob/latest/packages/components/psammead-paragraph/src/index.stories.jsx#L13

File: packages/components/psammead-caption/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 1.1.2 | [PR#498](https://github.com/bbc/psammead/pull/498) Update stories |
```
Comment (dr3) [inline packages/components/psammead-caption/CHANGELOG.md]:
```suggestion
| 1.1.2 | [PR#498](https://github.com/bbc/psammead/pull/498) Update stories to use new input provider |
```

File: packages/components/psammead-headings/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 1.1.3   | [PR#498](https://github.com/bbc/psammead/pull/498) Update stories |
```
Comment (dr3) [inline packages/components/psammead-headings/CHANGELOG.md]:
```suggestion
| 1.1.3   | [PR#498](https://github.com/bbc/psammead/pull/498) Update stories to use new input provider |
```

File: packages/components/psammead-inline-link/CHANGELOG.md [hunk 3]
```diff
@@ -1,17 +1,19 @@
...
-| 0.3.3   | [PR#323](https://github.com/bbc/psammead/pull/323) Update storybook badge url |
-| 0.3.2   | [PR#323](https://github.com/BBC/psammead/pull/323) Update readme storybook badge |
-| 0.3.1   | [PR#264](https://github.com/BBC/psammead/pull/319) Resolving package-lock issues. |
-| 0.3.0   | [PR#309](https://github.com/bbc/psammead/pull/309) Update colours to reflect new UX designs. |
-| 0.2.0   | [PR#248](https://github.com/BBC-News/psammead/pull/248) Remove top and buttom padding |
-| 0.1.3   | [PR#264](https://github.com/BBC/psammead/pull/264) Resolving package-lock issues. |
-| 0.1.2   | [PR#245](https://github.com/BBC-News/psammead/pull/245) Ensures documentation consistent across component packages. |
-| 0.1.1   | [PR#231](https://github.com/BBC-News/psammead/pull/231) Add link to Storybook to README |
-| 0.1.0   | [PR#106](https://github.com/BBC-News/psammead/pull/106) Create initial package with Inline Link component pulled in from [Simorgh](https://github.com/BBC-News/simorgh). |
+| 0.3.8 | [PR#498](https://github.com/bbc/psammead/pull/498) Update stories |
```
Comment (dr3) [inline packages/components/psammead-inline-link/CHANGELOG.md]:
```suggestion
| 0.3.8 | [PR#498](https://github.com/bbc/psammead/pull/498) Update stories to use new input provider |
```

File: packages/components/psammead-paragraph/CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 1.0.4   | [PR#498](https://github.com/bbc/psammead/pull/498) Update story |
```
Comment (dr3) [inline packages/components/psammead-paragraph/CHANGELOG.md]:
```suggestion
| 1.0.4   | [PR#498](https://github.com/bbc/psammead/pull/498) Update stories to use new input provider |
```

File: packages/components/psammead-sitewide-links/CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 0.3.7   | [PR#498](https://github.com/bbc/psammead/pull/498) Update stories |
```
Comment (dr3) [inline packages/components/psammead-sitewide-links/CHANGELOG.md]:
```suggestion
| 0.3.7   | [PR#498](https://github.com/bbc/psammead/pull/498) Update stories to use new input provider |
```

File: packages/components/psammead-story-promo/CHANGELOG.md [hunk 6]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
+| 0.1.3   | [PR#498](https://github.com/bbc/psammead/pull/498) Update stories. |
```
Comment (dr3) [inline packages/components/psammead-story-promo/CHANGELOG.md]:
```suggestion
| 0.1.3   | [PR#498](https://github.com/bbc/psammead/pull/498) Update stories to use new input provider |
```

File: packages/components/psammead-visually-hidden-text/CHANGELOG.md [hunk 7]
```diff
@@ -1,7 +1,9 @@
 # Psammead Visually Hidden Text Changelog
 
+<!-- prettier-ignore -->
 | Version | Description |
 | --------------------- |
+| 0.1.14  | [PR#498](https://github.com/bbc/psammead/pull/498) Update stories |
```
Comment (dr3) [inline packages/components/psammead-visually-hidden-text/CHANGELOG.md]:
```suggestion
| 0.1.14  | [PR#498](https://github.com/bbc/psammead/pull/498) Update stories to use new input provider |
```

**Review by dr3 — CHANGES_REQUESTED** [review]
LGTM after this :)

**Review by dr3 — APPROVED** [review]

**Comment by DenisHdz** [comment]
> We need to do Paragraph too, https://github.com/bbc/psammead/blob/latest/packages/components/psammead-paragraph/src/index.stories.jsx#L13

Does the `psammead-figure` need an update too?

**Review by dr3 — CHANGES_REQUESTED** [review]
Need to do Figure too, Good spot @DenisBBC

**Review by andrew-nowak — DISMISSED** [review]
lgtm :+1:

**Review by DenisHdz — DISMISSED** [review]
Great work 👍

**Review by dr3 — DISMISSED** [review]

File: packages/components/psammead-story-promo/src/index.stories.jsx [hunk 8]
```diff
@@ -20,10 +19,10 @@ const ImageComponent = (
...
 
 // eslint-disable-next-line react/prop-types
-const InfoComponent = ({ headlineText, summaryText }) => (
+const InfoComponent = ({ headlineText, summaryText, script }) => (
   <Fragment>
-    <Headline script={latin}>{headlineText}</Headline>
-    <Summary script={latin}>{summaryText}</Summary>
+    <Headline script={script}>{headlineText}</Headline>
+    <Summary script={script}>{summaryText}</Summary>
     <Timestamp datetime={text('Timestamp datetime', '2019-03-01T14:00+00:00')}>
```
Comment (DenisHdz) [inline packages/components/psammead-story-promo/src/index.stories.jsx]: The `script` prop should be also passed to the `Timestamp`.

**Review by DenisHdz — APPROVED** [review]

**Review by j-pendlebury — APPROVED** [review]
Looks good 👍

**Comment by jamesbrumpton** [comment]
LGTM. Happy for this to be merged.

**Pull request merged**